models.py

```diff
@@ -1252,6 +1252,13 @@ def get_prefs_for_emails(self, emails):
     return user_prefs
 
 
+class UserPrefForm(forms.Form):
+  notify_as_starrer = forms.BooleanField(
+      required=False,
```

Comment:
I'm curious.  It seems like one potential option here would be to make this field required and set a default to False.  Is there an advantage to making it optional vs required w/ a default?

Reply:
I don't know if the concept of required really makes sense for a checkbox because it is always either checked or unchecked, it cannot be undefined.   If I remove required=False then django adds a line of red text that says "This field is required." which seems awkward on that tiny page that has only one checkbox on it.